UefiDbgExt/swdebug.cpp

```diff
@@ -150,11 +150,36 @@ monitor (
   PCSTR           args
   )
 {
-  PSTR  Response;
+  PSTR         Response;
+  ULONG        Len;
+  const CHAR   *TruncateTag   = "#T#";
+  const ULONG  TruncateTagLen = sizeof ("#T#") - 1; // Exclude the null terminator.
+  ULONG        Offset;
 
   INIT_API ();
 
-  Response = MonitorCommandWithOutput (Client, args);
+  Offset = 0;
+  while (TRUE) {

```

Comment:
I think a generic doc as to why we loop would be good. I'd argue its fairly understandable from the code but it does not hurt to add more documentation.